modules/programs/fish.nix

```diff
@@ -99,18 +99,10 @@ in
 
   config = mkIf cfg.enable {
 
-    environment.etc."fish/foreign-env/shellInit".text = cfge.shellInit;
-    environment.etc."fish/foreign-env/loginShellInit".text = cfge.loginShellInit;
-    environment.etc."fish/foreign-env/interactiveShellInit".text = cfge.interactiveShellInit;
```

Comment:
This configuration is being set on the fish module itself, yet was being run through fenv. To me, it seems like if you're specifying init for fish specifically, it should be written in fish.